src/index.html

```diff
@@ -3,11 +3,55 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width,
+          user-scalable=no, initial-scale=1.0,
+          maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    >
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27&apos;&apos; Retina 5K Monoblock"
+        class="card__image"
+      >
+      <div class="card__title">
+        APPLE A1419 iMac 27&apos;&apos; Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__code">
+        Product code: 195434
+      </div>
+      <div class="card__stars">
+        <div class="stars stars--4">
+
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+
+        </div>
+        <div class="card__stars-count">
+          Reviews: 5
+        </div>
+      </div>
+      <div class="card__price">
+        <div class="card__price-tag">
+          Price:
+        </div>
+        <div class="card__price-value">
+          $2,199
+        </div>
```

Comment:
```suggestion
        <div class="card__price-text">
          Price:
        </div>
        <div class="card__price-value">
          $2,199
        </div>
```
those are also p or span tags